init.sql

```diff
@@ -109,7 +109,8 @@ create table asset
     chain_data_id         bigint references asset_data (id),
     -- visibility
     created_at            timestamp with time zone     default (now() at time zone 'utc'),
-    burnt                 bool                not null default false
+    burnt                 bool                not null default false,
+    seq                   bigint              not null
```

Comment:
I'm not sure exactly what you mean, wondering if you are asking me to create some kind of group called "compression" but the only group concept I know of for SQL is user groups and `GROUP BY` statement.

Are you asking me to create another table or just make `burnt` able to be NULL since it will be NULL for uncompressed?

Reply:
oh that is simpler request than i thought 😄

Reply:
Done